src/middlewares/normalizeCategoryQueryParams.js

```
@@ -0,0 +1,24 @@
+
+const normalizeCategoryQueryParams = (req, res, next) => {
+  const { limit, skip, sort, categories } = req.query
+
+  req.query.limit = Number.isInteger(Number(limit)) ? Number(limit) : DEFAULT_CATEGORY_LIMIT
+
+  req.query.skip = Number.isInteger(Number(skip)) ? Number(skip) : DEFAULT_CATEGORY_SKIP
+
+  req.query.sort = sort?.orderBy ? { [sort.orderBy]: sort.order?.toLowerCase() === 'desc' ? -1 : 1 } : {}
+
+  const categoryArr =
+    categories && (Array.isArray(categories) ? categories : categories.split(',').map((s) => s.trim()))
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)